blazar/manager/enforcement.py

```diff
@@ -153,6 +144,35 @@ def remove_lease_exception(self, user_name):
     def setup_usage_enforcement(self, project_name):
         self.initialize_project_allocation(project_name)
 
+    def check_user_max_leases(self, lease_values, resource_type):
+        user_leases = [
+            l for l in db_api.lease_get_all_by_user(lease_values['user_id'])
+            if l['status'] in [status.lease.ACTIVE, status.lease.PENDING]]
+
+        leases_by_resource_type = [
+            l['id'] for l in user_leases
+            if any([
+                x['resource_type'] == resource_type for x
+                in l['reservations']])
+        ]
+
+        existing = len(leases_by_resource_type)
+        max = CONF.enforcement.default_max_leases_by_resource_type.get(
```

Comment:
Is `max` a reserved word? I don't know how Python handles that. It seems to still be ok in Python3. Maybe `max_leases` is better.

Reply:
No, max() is a built-in function so the code will still work, but it's better practice to use max_leases so I changed.